definitions/family-private/json/CaseEventToFields.json

```diff
@@ -78,5 +78,108 @@
     "PageDisplayOrder": 1,
     "PageFieldDisplayOrder": 6,
     "FieldShowCondition": "ApplicantHasValidReasonForNotAttendingMiam=\"Yes\""
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseEventID": "specialMesures",
+    "CaseFieldID": "LabelSpecialMesuresRequired",
+    "DisplayContext": "READONLY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 1,
+    "ShowSummaryChangeOption": "Yes"
```

Comment:
are you sure you need it for this field? It's a read-only label

Reply:
it will display the label in summary if the option is set to "Yes" - could be designed like that